bazaar/rpc/src/lib.rs

```diff
@@ -56,6 +56,7 @@ where
 pub struct BazaarRpc<Client, Block, AccountId> {
 	client: Arc<Client>,
 	_marker: std::marker::PhantomData<(Block, AccountId)>,
+	#[allow(unused)]
 	deny_unsafe: DenyUnsafe,
```

Comment:
I doubt that we are ever going to use this flag again. I know this is a breaking API change, but I favour removing it now.